config/parser.go

```diff
@@ -0,0 +1,74 @@
+package config
+
+import (
+    "io/ioutil"
+    "encoding/json"
+
+    "./../error"
+)
+
+const path = "./config.json"
+
+type Config struct {
+    Db *Db
+    Email *Email
+    Sreality *Sreality
+}
+
+type Db struct {
+    Engine string
+    Host string
+    Port int
+    Username string
+    Password string
+    Database string
+}
+
+type Email struct{
+    To string
+    From string
+    Server string
+    Tls bool
+    TlsPort int `json:"tls_port"`
+    Username string
+    Password string
+}
+
+type Sreality struct{
+    RealityType int `json:"reality_type"`
+    OperationType int `json:"operation_type"`
+    RealityOptions []int `json:"reality_options"`
+    Country int
+    Region []int
+    District []int
+    PageResults int `json:"page_results"`
+    EstateAge int `json:"estate_age"`
+    Square *rangeInt
+    Price *rangeInt
+}
+
+type rangeInt struct {
+    Min int
+    Max int
+}
+
+func New() (*Config) {
+    config := new(Config);
+    content, err := ioutil.ReadFile(path);
+    err = json.Unmarshal(content, config)
+    error.DebugError(err)
```

Comment:
AlWAYS return an error, it's the responsibility of the caller to handle that error. Call method not just `New()`, but `NewConfig()`.

You can use pattern `MustNewConfig()` in that case, it's ok to `panic()` in that method, if further program execution doesn't make sense.